core/src/main/java/org/apache/accumulo/core/client/rfile/RFileWriter.java

```diff
@@ -250,5 +259,22 @@ public void append(Iterable<Entry<Key,Value>> keyValues) throws IOException {
   @Override
   public void close() throws IOException {
     writer.close();
+    loadPlanCollector.close();
+  }
+
+  /**
+   * If no split resolver was provided when the RFileWriter was built then this method will return a
+   * simple load plan of type {@link org.apache.accumulo.core.data.LoadPlan.RangeType#FILE} using
+   * the first and last row seen. If a splitResolver was provided then this will return a load plan
+   * of type {@link org.apache.accumulo.core.data.LoadPlan.RangeType#TABLE} that has the split
+   * ranges the rows written overlapped.
+   *
+   * @param filename
```

Comment:
The `@param filename` javadoc entry needs a description.

Reply:
done in 926dec7fc5ef26fd1107491075b9270f17fc3d1c